src/components/ha-climate-control.html

```diff
@@ -0,0 +1,104 @@
+<link rel='import' href='../../bower_components/polymer/polymer.html'>
+<link rel="import" href="../../bower_components/iron-flex-layout/iron-flex-layout-classes.html">
+<link rel="import" href="../../bower_components/paper-icon-button/paper-icon-button.html">
+
+
+<dom-module id="ha-climate-control">
+  <template>
+    <style is="custom-style" include="iron-flex iron-flex-alignment"></style>
+    <style>
+      :host {
+        display: flex;
+      }
+      /* local DOM styles go here */
+      .container-ha-climate-control {
+        width: 100%;
+        @apply(--layout-flex);
+        @apply(--layout-horizontal);
+        @apply(--layout-justified);
+      }
+      iron-icon {
+        fill: var(--ha-climate-control-color, rgba(0,0,0,0));
+        stroke: var(--ha-climate-control-outline-color, currentcolor);
+      }
+      .target-temperature {
+        @apply(--layout-self-center);
+        font-size: 200%;
+      }
+      .control-buttons {
+        font-size: 200%;
+        text-align: right;
+      }
+      :host([pressed]) iron-icon {
+        fill: var(--ha-climate-control-pressed-color, currentcolor);
+      }
+
+      paper-icon-button {
+        height: 48px;
+        width: 48px;
+      }
+    </style>
+
+    <!-- local DOM goes here -->
+    <div class="container-ha-climate-control">
+      <div class="target-temperature">
+        [[value]] [[units]]
+      </div>
+      <div class="control-buttons">
+        <div>
+          <paper-icon-button icon="mdi:chevron-up" on-tap="incrementValue"></paper-icon-button>
+        </div>
+        <div>
+          <paper-icon-button icon="mdi:chevron-down" on-tap="decrementValue"></paper-icon-button>
+        </div>
+      </div>
+    </div>
+  </template>
+  <script>
+    Polymer({
+      is: 'ha-climate-control',
+      properties: {
+        value: {
+          type: Number,
+          notify: true,
+          reflectToAttribute: true,
+          observer: 'valueChanged'
+        },
+        units: {
+          type: String,
+          reflectToAttribute: true,
+        },
+        min: {
+          type: Number,
+          reflectToAttribute: true,
+        },
+        max: {
+          type: Number,
+          reflectToAttribute: true,
+        },
+        step: {
+          type: Number,
+          value: 1,
+          reflectToAttribute: true,
+        },
+      },
+      incrementValue: function () {
+        var newval = this.value + this.step;
+        if (newval <= this.max) {
```

Comment:
if this step is 0.5 in those circumstances we should end up with 11, but I can also just set us to max under those scenarios.